homeassistant/bootstrap.py

```diff
@@ -186,6 +186,22 @@ def from_config_file(config_path, hass=None):
 def enable_logging(hass):
     """ Setup the logging for home assistant. """
     logging.basicConfig(level=logging.INFO)
+    try:
+        from colorlog import ColoredFormatter
+        logging.getLogger().handlers[0].setFormatter(ColoredFormatter(
+            "%(log_color)s%(asctime)s (%(threadName)s) [%(name)s] %(message)s%(reset)s",
```

Comment:
I personally don't care which thread is involved. But I would like to see the message type because after copying the output away from the terminal the colors are gone.  

And perhaps it's worth to use the same format for the date/time as in the log file.

Just my 0.2 cents.